hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/ConditionalWriteLockConfig.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.client.transaction.lock;
+
+import org.apache.hudi.common.config.ConfigProperty;
+import org.apache.hudi.common.config.HoodieConfig;
+import org.apache.hudi.common.config.LockConfiguration;
+import org.apache.hudi.common.config.TypedProperties;
+
+import org.apache.hadoop.fs.Path;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.apache.hudi.common.config.HoodieCommonConfig.BASE_PATH;
+import static org.apache.hudi.client.transaction.lock.ConditionalWriteLockProvider.DEFAULT_TABLE_LOCK_FILE_NAME;
+import static org.apache.hudi.common.table.HoodieTableMetaClient.LOCKS_FOLDER_NAME;
+
+public class ConditionalWriteLockConfig extends HoodieConfig {
+  private static final String SINCE_VERSION_1_0_2 = "1.0.2";
+  private static final String CONDITIONAL_WRITE_LOCK_PROPERTY_PREFIX = LockConfiguration.LOCK_PREFIX
+      + "conditional_write.";
+  public static final ConfigProperty<String> LOCK_INTERNAL_STORAGE_LOCATION = ConfigProperty
+      .key(CONDITIONAL_WRITE_LOCK_PROPERTY_PREFIX + "locks_location")
+      .defaultValue("")
+      .markAdvanced()
+      .sinceVersion(SINCE_VERSION_1_0_2)
+      .withDocumentation(
+          "For conditional write based lock provider, the optional URI where lock files are written. "
+              + "Must be the same filesystem as the table path and should conditional writes. "
+              + "By default, writes to " + LOCKS_FOLDER_NAME + Path.SEPARATOR
+              + DEFAULT_TABLE_LOCK_FILE_NAME + ".json under the table base path.");
+
+  public static final ConfigProperty<Long> LOCK_VALIDITY_TIMEOUT_MS = ConfigProperty
+      .key(CONDITIONAL_WRITE_LOCK_PROPERTY_PREFIX + "lock_validity_timeout_ms")
+      .defaultValue(TimeUnit.MINUTES.toMillis(5))
```

Comment:
If the value should never be below 1s, should the unit be "second" so it's easier to config and read?